zilencer/management/commands/populate_db.py

```diff
@@ -204,9 +205,12 @@ def handle(self, **options):
             zulip_realm_bots = [
                 ("Zulip New User Bot", "new-user-bot@zulip.com"),
                 ("Zulip Error Bot", "error-bot@zulip.com"),
+                ("Zulip Webhook Bot", "webhook-bot@zulip.com"),
                 ]
             zulip_realm_bots.extend(all_realm_bots)
-            create_users(realms, zulip_realm_bots, bot=True)
+            create_users(realms, zulip_realm_bots, bot_type=UserProfile.DEFAULT_BOT)
+            webhook_bot = UserProfile.objects.get(email='webhook-bot@zulip.com')
+            do_change_bot_type(webhook_bot, UserProfile.INCOMING_WEBHOOK_BOT)
```

Comment:
can we just do two create_users calls, the second being `create_users(realms, zulip_realm_webhook_bots, bot_type=UserProfile.INCOMING_WEBHOOK_BOT` ?